server/svix-server/src/core/security.rs

```diff
@@ -114,10 +209,32 @@ where
     }
 }
 
-pub fn generate_token(keys: &Keys, org_id: OrganizationId) -> Result<String> {
-    let claims = Claims::create(Duration::from_hours(24 * 365 * 10))
+pub fn generate_token(
```

Comment:
Split (or make it private and wrap it) to two functions:
* generate_org_tokne
* generate_app_token

Because otherwise it's not clear (when calling this function) what's the level of access.